--- a/src/threads/mbedtls.c
+++ b/src/threads/mbedtls.c
@@ -56,7 +56,7 @@
     fprintf(stderr,
             "MbedTLS needs to have threading enabled with "
             "MBEDTLS_THREADING_PTHREAD or MBEDTLS_THREADING_ALT "
             "in mbedtls_config.h\n");
-#warn "MbedTLS needs to have threading enabled with " \
+#warning "MbedTLS needs to have threading enabled with " \
       "MBEDTLS_THREADING_PTHREAD or MBEDTLS_THREADING_ALT in mbedtls_config.h"
     return SSH_ERROR;
 #endif
